qdrant-landing/content/articles/vector-search-resource-optimization.md

```diff
@@ -0,0 +1,576 @@
+
+You can control the number of extra candidates by setting an `oversampling` parameter. For example, if your desired number of results (`limit`) is 4 and you set an `oversampling` factor of 2, Qdrant will retrieve 8 candidates (4 × 2).
+
+You can adjust the oversampling factor to control how many extra vectors Qdrant includes in the initial pool. More candidates mean a better chance of obtaining high-quality top-K results, especially after rescoring with the original vectors.
+
+---
+Learn more about [**Oversampling**](/articles/what-is-vector-quantization/#2-oversampling).
+
+---
+
+#### Rescoring
+
+After oversampling to gather more potential matches, each candidate is re-evaluated based on additional criteria to ensure higher accuracy and relevance to the query.
+
+The rescoring process maps the quantized vectors to their corresponding original vectors, allowing you to consider factors like context, metadata, or additional relevance that wasn’t included in the initial search, leading to more accurate results.
+
+**Example of Rescoring and Oversampling:**:
+
+```python
+client.query_points(
+    collection_name="my_collection",
+    query_vector=[0.22, -0.01, -0.98, 0.37],
+    search_params=models.SearchParams(
+        quantization=models.QuantizationSearchParams(
+            rescore=True,   # Enables rescoring with original vectors
+            oversampling=2  # Retrieves extra candidates for rescoring
+        )
+    ),
+    limit=4  # Desired number of final results
+)
+```
+---
+Learn more about [**Rescoring**](/articles/what-is-vector-quantization/#3-rescoring-with-original-vectors).
+
+---
+
+#### Reranking
+
+Reranking adjusts the order of search results based on additional criteria, ensuring the most relevant results are prioritized. 
+
+This method is about taking the results from different search methods and reordering them based on some additional processing using the content of the documents, not just the scores. This processing may rely on an additional neural model, such as a cross-encoder which would be inefficient enough to be used on the whole dataset. 
+
+<img src="/articles_data/vector-search-resource-optimization/reranking.png" alt="reranking" style="width: 75%;">
+
+These methods are practically applicable only when used on a smaller subset of candidates returned by the faster search methods. Late interaction models, such as ColBERT, are way more efficient in this case, as they can be used to rerank the candidates without the need to access all the documents in the collection.
+
+**Example:**
+
+```python
+client.query_points(
+        "collection-name",
+       prefetch=prefetch, # Previous results
+       query=late_vectors, # Colbert converted query
+       using="colbertv2.0",
+       with_payload=True,
+       limit=10,
+)
+```
```

Comment:
```suggestion
```python
client.query_points(
       "collection-name",
       prefetch=prefetch, # Previous results
       query=late_vectors, # Colbert converted query
       using="colbertv2.0",
       with_payload=True,
       limit=10,
)
```
```

Just the formatting.